### PR TITLE
Add appname and control logs (WIP)

### PR DESCRIPTION
**Pull request by blogh**
In response to : https://github.com/dalibo/pg_activity/issues/158
I have added application name and an option to disable query logging for pg_activity.

### CODE REVIEW

**Comment by dlax** [comment]
About application_name, I think we can define this when connecting with psycopg: `psycopg2.connect(..., application_name="pg_activity")`.

**Comment by blogh** [comment]
I was wondering if the approach using queries is correct so I tried using dsn's like :

```
application_name=pg_activity options='-c log_min_duration_statement=-1'
```

The problem being that some parameters are version dependant which requires a connection to be aware of.

**Comment by blogh** [comment]
Just read your message, will try it your way.

**Comment by blogh** [comment]
Indeed ... Better ...

File: pg_activity [hunk 1]
```diff
@@ -261,6 +261,14 @@ def get_parser() -> ModifiedOptionParser:
         help="Disable App",
         default=False,
     )
+    # --hide-queries-in-logs
+    group.add_option(
+        "--hide-queries-in-logs",
+        dest="hidequeriesinlogs",
```
Comment (dlax) [inline pg_activity:267]: This would be more readable as `hide_queries_in_logs`.

**Review by dlax — APPROVED** [review]
LGTM, modulo the nit about option name.

When merging, I'd suggest to squash commits to have a clean history and write a meaningful commit message. Thanks.

File: pg_activity (hunk 1 above)
Comment (blogh) [reply pg_activity:267]: I agree, I used this because all the other option names where squashed like this.
Will used your format, then.

Comment (blogh) [reply pg_activity]: done

**Pull request merged**